providers/provider_data.go

```diff
@@ -283,7 +284,12 @@ func (p *ProviderData) buildSessionFromClaims(rawIDToken, accessToken string) (*
 }
 
 func (p *ProviderData) getClaimExtractor(rawIDToken, accessToken string) (util.ClaimExtractor, error) {
```

Comment:
Regarding testing. Could you take some time a extend the test case `TestProviderData_buildSessionFromClaims` inside the `provider_data_test.go`

We should be able to set the new skip option inside the tests and get a valid response:
https://github.com/oauth2-proxy/oauth2-proxy/blob/63727103db4a8bb22872e8e82cb5fe38018ffbad/providers/provider_data_test.go#L427-L441

Reply:
Thanks, I will take a look.

Reply:
Testing SkipClaimsFromProfileURL turned out to be a bit trickier than expected. The only feasible way I can see, is to use httptest.Server, set ProfileURL (and AccessToken + getAuthorizationHeaderFunc), and test if the server receives a request based on the value for SkipClaimsFromProfileURL. Ref commit 7dc8c14d51276a206dd8d13dcc20977714bcae42

Please let me know if you have any better ideas or suggestions.